pkg/apis/sessions/session_state_test.go

```diff
@@ -353,3 +274,24 @@ func TestIntoEncryptAndIntoDecrypt(t *testing.T) {
 		})
 	}
 }
+
+func compareSessionStates(t *testing.T, left *SessionState, right *SessionState) {
```

Comment:
Nice extraction to a helper 👍

Reply:
Thanks! Those pesky Time based fields made a pure object compare not work as I had hoped.

Reply:
Rather than left, right, we should call these expected and got? Or expected and actual? That would tie up with the assert right? I think in assert the parameter order is `t testing.T, expected interface{}, actual interface{}`?

Reply:
At one point I use this helper to compare if taking the encode/decode route with 2 different paths still results in sessions states that are equal:

`compareSessionStates(t, decoded, decodedCompressed)`

That is the only reason I didn't use the expected terminology.  But I suppose I am expecting the decoded without compression to equal the decoded with compression -- so I can switch to that naming convention